pkg/cmd/controller/controller_build.go

```diff
@@ -1361,10 +1362,12 @@ func toScmStatusAndDescription(activity *v1.PipelineActivity) (string, string) {
 	}
 	stagesByStatus := activity.StagesByStatus()
 
+	runningStages := ""
 	if len(stagesByStatus[v1.ActivityStatusTypeRunning]) > 0 {
+		runningStages = strings.Join(stagesByStatus[v1.ActivityStatusTypeRunning], ",")
 		description = fmt.Sprintf("Pipeline running stage(s): %s", strings.Join(stagesByStatus[v1.ActivityStatusTypeRunning], ", "))
 	}
-	return scmStatus, description
+	return scmStatus, description, runningStages
```

Comment:
how long does this _runningStages_ get? Is there a risk that this exceed the 64 char limit?

Reply:
I might the stages as array/slice!?

Reply:
So k8s annotations can be absurdly long - 256kb, if I remember correctly - so this isn't a concern. =)